packages/e2e-tests/src/specs/editor/shopping/woocommerce.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies -- in package.json
+import { matchers } from 'jest-json-schema';
+
+import {
+  createNewStory,
+  withExperimentalFeatures,
+  publishStory,
+  activatePlugin,
+  deactivatePlugin,
+  takeSnapshot,
+} from '@web-stories-wp/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import * as schema from './schema.json';
+
+// eslint-disable-next-line jest/require-hook -- matchers only for this set of tests
+expect.extend(matchers);
+
+describe('Shopping', () => {
+  describe('Shopping schema', () => {
+    const WPVersion = process.env?.WP_VERSION || 'latest';
+
+    withExperimentalFeatures(['shoppingIntegration']);
+
+    beforeAll(async () => {
+      if ('latest' === WPVersion) {
+        await activatePlugin('woocommerce');
+      }
+    });
+
+    afterAll(async () => {
+      if ('latest' === WPVersion) {
+        await deactivatePlugin('woocommerce');
+      }
+    });
+
+    if (WPVersion !== 'latest') {
+      //eslint-disable-next-line jest/expect-expect, jest/no-focused-tests
+      it.only('minimum WordPress requirement not met', () => {});
+    }
+
+    it('should should match a valid schema', async () => {
+      await createNewStory();
+      await expect(page).toClick('[aria-controls="library-pane-shopping"]');
+      await expect(page).toClick('[aria-label="Product"]');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('Enter');
+      await expect(page).toClick('button', { text: 'Insert product' });
+      await takeSnapshot(page, 'insert product');
```

Comment:
I think we can introduce this snapshot later once we add e2e tests for product insertion in general, not for the schema test.